Tools/autotest/default_params/copter-optflow.parm

```diff
@@ -1,6 +1,8 @@
-ARMING_CHECK -33
-EKF_GPS_TYPE 3
+ARMING_CHECK 786390
 EK2_GPS_TYPE 3
```

Comment:
why keep this one when we are on EKF3 ?

Reply:
Yes, indeed, we could remove these parameters that are already set to these values by default.